### PR TITLE
Roll BraveSearchAdStudy for 100% on iOS in Release (prod)

### PR DESCRIPTION
**Pull request by aseren**
Resolves https://github.com/brave/brave-variations/issues/1012

This was already tested for Desktop and Android for a very long time.

PR to the main: https://github.com/brave/brave-variations/pull/1010

### CODE REVIEW

**Review by tmancey — APPROVED** [review]
LGTM++

File: seed/seed.json [hunk 1]
```diff
@@ -1888,7 +1888,8 @@
             "filter": {
                 "channel": [
                     "NIGHTLY",
-                    "BETA"
+                    "BETA",
+                    "IOS"
```
[the remark this answers is not in the document]
Comment (aseren) [reply seed/seed.json:1892]: Thanks for finding this, this should be RELEASE, as in staging PR: https://github.com/brave/brave-variations/pull/1010/files

[the remark this answers is not in the document]
Comment (iambrianfung) [reply seed/seed.json]: https://github.com/brave/brave-variations/pull/1011/commits/f929697073312d2847cae40afd77a5d72eb3b0a8

**Review by tmancey — APPROVED** [review]
LGTM++

**Review by iambrianfung — APPROVED** [review]

**Review by LaurenWags — APPROVED** [review]
Uplift to `production` is approved 👍🏻 QA has verified the PR on `main` as per https://github.com/brave/brave-variations/pull/1011#issuecomment-2085343013.

**Pull request merged**